ChaosMod/Effects/db/Player/PlayerSetIntoRandomVeh.cpp

```diff
@@ -12,9 +12,7 @@ static void OnStart()
 	for (Vehicle veh : GetAllVehs())
 	{
 		Vector3 vehPos = GET_ENTITY_COORDS(veh, false);
-		bool useGroundZ = GET_GROUND_Z_FOR_3D_COORD(vehPos.x, vehPos.y, vehPos.z, &groundZ, false, false);
-		bool vehCollision = HAS_COLLISION_LOADED_AROUND_ENTITY(veh);
-		if (veh != playerVeh && useGroundZ && vehCollision)
+		if (((veh != playerVeh) && (GET_GROUND_Z_FOR_3D_COORD(vehPos.x, vehPos.y, vehPos.z, &groundZ, false, false))) && HAS_COLLISION_LOADED_AROUND_ENTITY(veh))
```

Comment:
here too, no need to make additional brackets if you compare everything with AND
-> `if (veh != playerVeh && GET_GROUND_Z_FOR_3D_COORD(vehPos.x, vehPos.y, vehPos.z, &groundZ, false, false) && HAS_COLLISION_LOADED_AROUND_ENTITY(veh))`